src/drt/test/ispd18_sample.defok

```diff
@@ -27,8 +27,8 @@ TRACKS X 83800 DO 52 STEP 400 LAYER Metal2 ;
 TRACKS Y 72010 DO 51 STEP 380 LAYER Metal2 ;
 TRACKS X 83800 DO 52 STEP 400 LAYER Metal1 ;
 TRACKS Y 72010 DO 51 STEP 380 LAYER Metal1 ;
-GCELLGRID X 83600 DO 18 STEP 5700 ;
-GCELLGRID Y 71820 DO 16 STEP 5700 ;
+GCELLGRID X 83600 DO 3 STEP 6000 ;
+GCELLGRID Y 71820 DO 3 STEP 5700 ;
```

Comment:
18->3 is a surprisingly large change - is this correct?

Reply:
Yes, it is correct because previously, GRT was not considering the origin of the design to calculate the number of GCells. Since this design has a non-zero origin, the resulting value is much larger than it should be.